deploy/osd-pcap-collector/06-pcap-dedicated-admins-ClusterRoleBind.yaml

```diff
@@ -1,11 +0,0 @@
-apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
This ClusterRoleBinding needs to remain to allow customers to run pcaps as dedicated admin.